features/fixtures/bakingtesttransform.py

```diff
@@ -0,0 +1,70 @@
+# ---
```

Comment:
not sure what this file is, do we still need it?

Reply:
This was to test the homemade dataset which displays all the direction qualities we wanted to test. We don't need it as the transform code is brought into baking.feature